app.js

```diff
@@ -1,9 +1,11 @@
-for (i = 1; i <= 99; i++){
-    if ( i % 3 == 0 && i % 5 == 0 ) {
-        console.log('FizzBuzz' + i)
-    }  else if ((i % 3)==0) {
-        console.log('Fizz' + i);
-    }  else if ((i % 5)==0) {
-        console.log('Buzz' + i);
+const Name = prompt("日本の首都は？");
+
+while (Name.includes("")) {
```

Comment:
条件が必ずtrueになれば良いので、ここはtrueをそのまま入れてしまう方がスマートかなと思います。
```suggestion
while (true) {
```